### PR TITLE
feat: disable add to contest if an item is out of bounds

### LINKED ISSUE #309: Disable contest button on entity out of bounds
Disable contest button on entity out of bounds

### PR DESCRIPTION
**Pull request by cazala**
Closes #309 

Screen Shot 2019-03-11 at 2 17 51 PM

### CODE REVIEW

**Comment by nicosantangelo** [comment]
If I have an item out of bounds **and** too many triangles this happens:

pip

File: src/modules/translation/languages/en.json [hunk 1]
```diff
@@ -71,6 +71,7 @@
   "topbar": {
     "add_to_contest": "Add to contest",
     "add_to_contest_disabled_limits": "Too many {metric}{br}Try deleting some items!",
+    "add_to_contest_disabled_bounds": "Item out of boundaries{br}Try to move it in!",
```
Comment (nicosantangelo) [inline src/modules/translation/languages/en.json]: We might need to change this copy, not sure if `"move it in!"` is entirely correct @twaldorf

Comment (twaldorf) [reply src/modules/translation/languages/en.json]:
```suggestion
    "add_to_contest_disabled_bounds": "Item out of bounds{br}Move the item inside the boundaries!",
```

Is that too long?

Comment (belohlavek) [reply src/modules/translation/languages/en.json]: Is this the only thing blocking?

Comment (cazala) [reply src/modules/translation/languages/en.json]: done ✅

**Pull request merged**